src/rng.cpp

```diff
@@ -57,6 +57,10 @@ double rng_exponential( double min, double mean )
 
 bool one_in( int chance )
 {
+    if( chance < 0 ) {
+        debugmsg( "Invalid one_in: %d", chance );
```

Comment:
i don't agree that this should change current behavior, i also think that this debug message will cause more toruble than it's worth.

Reply:
( Well this is just an optional addon... Will revert it if you say so.

Reply:
please do